src/panels/lovelace/cards/energy/hui-energy-solar-graph-card.ts

```diff
@@ -96,6 +97,13 @@ export class HuiEnergySolarGraphCard
             )}
             chart-type="bar"
           ></ha-chart-base>
+          ${!this._chartData.datasets.length
+            ? html`<div class="no-data">
+                ${this._start.getTime() > Date.now()
```

Comment:
Shouldn't we only show the data to arrive message if we test true with the `isToday(this._start)`